package.json

```diff
@@ -6,13 +6,13 @@
   ],
   "binary": {
         "module_name" : "xmljs",
-        "module_path" : "./build/Release/",
+        "module_path" : "./lib/binding/{node_abi}-{platform}-{arch}",
         "host"        : "https://github.com",
         "remote_path" : "./libxmljs/libxmljs/releases/download/v{version}/",
         "package_name": "{node_abi}-{platform}-{arch}.tar.gz"
   },
   "description": "libxml bindings for v8 javascript engine",
-  "version": "0.18.4",
+  "version": "0.18.5",
```

Comment:
I'm not sure about your policy to bump the version number, but I had to increment it so that a fresh npm install would not try again to pull it down from github